hoprd/keypair/src/key_pair.rs

```diff
@@ -219,17 +200,19 @@ impl PartialEq for HoprKeys {
     }
 }
 
-impl HoprKeys {
+impl Randomizable for HoprKeys {
```

Comment:
This could be automatically derivable to minimize the code.

Reply:
Feel free to write a proc macro for it.